astrolabe/commands.py

```diff
@@ -22,6 +22,16 @@
 LOGGER = logging.getLogger(__name__)
 
 
+def get_organization_by_id(*, client, org_id):
+    """Get the organization by the given id `org_id`."""
+    org = client.orgs[org_id].get().data 
+    LOGGER.debug("Organization details: {}".format(org))   
+    return org    
+
+
+# TODO(etsai) - this method is inefficient, DO NOT USE.  
+# We have thousands of organizations that we'd have to page through 
+# and there's the risk of naming collision
 def get_one_organization_by_name(*, client, organization_name):
```

Comment:
Is there any reason to keep this function around? If we shouldn't use it, we should remove it.

Reply:
will remove